contracts/round/RoundFactory.sol

```diff
@@ -90,12 +90,8 @@ contract RoundFactory is IRoundFactory, OwnableUpgradeable {
    * @notice Clones RoundImplementation a new round and emits event
    *
    * @param encodedParameters Encoded parameters for creating a round
-   * @param ownedBy Program which created the contract
    */
-  function create(
-    bytes calldata encodedParameters,
-    address ownedBy
```

Comment:
This is now being replaced by msg.sender. 
So there is no necessity that only a program should be allowed to create a round. We could enforce this but I don't see much value in that as all we are doing is emitting and event